src/script/common/c_content.cpp

```diff
@@ -275,6 +275,7 @@ void read_object_properties(lua_State *L, int index,
 	}
 	lua_pop(L, 1);
 	getboolfield(L, -1, "backface_culling", prop->backface_culling);
+	getintfield(L, -1, "glow", prop->glow);
```

Comment:
I think you may need to also add this to `push_object_properties()` L302 below.

Reply:
I did this when i added the 'can_zoom' property, see https://github.com/minetest/minetest/commit/561a01cc2a8ca0b83b37975a048d34dcfd0d41e1

Reply:
Yes it's needed for:
```
// get_properties(self)
int ObjectRef::l_get_properties(lua_State *L)
{
	NO_MAP_LOCK_REQUIRED;
	ObjectRef *ref = checkobject(L, 1);
	ServerActiveObject *co = getobject(ref);
	if (co == NULL)
		return 0;
	ObjectProperties *prop = co->accessObjectProperties();
	if (!prop)
		return 0;
	push_object_properties(L, prop);
	return 1;
}
```